docs/_book.yaml

```diff
@@ -23,6 +23,8 @@ upper_tabs:
         path: /tensorboard/graphs
       - title: "Hyperparameter tuning"
         path: /tensorboard/hyperparameter_tuning_with_hparams
+      - title: "Projector plugin"
```

Comment:
We want to avoid using "plugin" in user documentation. How about "Embedding projector" or "Projector dashboard"?